stripe/res/values/strings.xml

```diff
@@ -106,4 +106,5 @@
     <string name="no_payment_methods">No payment methods.</string>
     <!--Text informing user they should add a debit or credit card. Shown when there are no existing payment methods in the application-->
     <string name="add_card">Add a new debit or credit card to make purchases in this app.</string>
+    <string name="invalid_shipping_information">Invalid Shipping Address</string>
```

Comment:
Do we need to add this to the translation request?

Reply:
(sort of, i *think* I added it but was planning on grabbing the strings from the iOS codebase)